### PR TITLE
Support app_logo_dark

### LINKED ISSUE #1003: Support dark mode app logo

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #1003

### CODE REVIEW

File: resources/authgear/templates/en/web/__header.html [hunk 1]
```diff
@@ -1,11 +1,20 @@
 {{ define "__header.html" }}
+
+{{ $light := call $.StaticAssetURL "app-logo" }}
+{{ $dark := call $.StaticAssetURL "app-logo-dark" }}
+{{ if not $dark }}
```
Comment (carmenlau) [inline resources/authgear/templates/en/web/__header.html:5]: I think we may not need js for that this feature? And only assign ` $dark = $light` when `not $dark or not $.DarkThemeEnabled`?

Comment (louischan-oursky) [reply resources/authgear/templates/en/web/__header.html:5]: Updated the approach to not use JS at all.

**Comment by louischan-oursky** [comment]
The ResourceNotFound should be fixed.

File: resources/authgear/templates/en/web/__header.html [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+{{ $dark = $light }}
+{{ end }}
+
+{{ if $.DarkThemeEnabled }}
+<img class="banner light" src="{{ $light }}">
+<img class="banner dark" src="{{ $dark }}">
+{{ else }}
+<img class="banner" src="{{ $light }}">
```
Comment (carmenlau) [inline resources/authgear/templates/en/web/__header.html:13]: 💯

**Review by carmenlau — APPROVED** [review]

**Pull request merged**